lib/modules/mod_pinp.phtml

```diff
@@ -1347,6 +1352,13 @@ class pinp {
 						$i++;
 					}
 				}
+				if ( $i > 0 ) {
+				//	echo "hier:\n";
+				//	echo $result . "\n";
+				//	echo "----\n";
+					$result = "($result ?? null)!!!!";
```

Comment:
`!!!!` doesn't look right...

Reply:
Understandable - the !!!! seems to be a marker that is used in the code further on, so it actually makes sense to have it there.